streams/src/test/java/org/apache/kafka/streams/state/internals/RocksDBSegmentedBytesStoreTest.java

```diff
@@ -52,6 +51,7 @@
 import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertTrue;
 
+// TODO: this test does not cover time window serdes
```

Comment:
should we better create a JIRA for this -- a TODO might get lost

Reply:
Ack.